.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Pull Request
+about: Fix an issue or add a new feature.
+---
+
+**Checklist for this pull request**
+
+<!--
+    Thank you for your contribution! Before submitting this PR, please
+    make sure that:
+-->
+
+- [ ] The new code additions passed the tests (`npm test`).
+- [ ] The linter ran and found no issues (`npm run-script lint`).
```

Comment:
Maybe we can add a ligne here with something like "Running `./node_modules/.bin/tslint -c tslint.json 'src/**/*.ts' --fix` will fix most of the linting problems automatically", or something like that?

Reply:
Good idea, maybe it's worth adding a new script named: "lint:fix" to wrap this command as part of the PR?

Reply:
Yep, good idea! I just realized `npm run lint -- --fix` also works, but `lint:fix` is probably less confusing than the quadruple dash 😁.